
Dont scare people with something that should be an informal message.

Signed-off-by: Thomas Backlund <tmb@mageia.org>

--- linux/arch/x86/mm/pti.c.orig
+++ linux/arch/x86/mm/pti.c
@@ -388,16 +388,14 @@ void __init pti_init(void)
 #ifdef CONFIG_X86_32
 	if (boot_cpu_has(X86_FEATURE_PCID)) {
 		/* Use printk to work around pr_fmt() */
-		printk(KERN_WARNING "\n");
-		printk(KERN_WARNING "************************************************************\n");
-		printk(KERN_WARNING "** WARNING! WARNING! WARNING! WARNING! WARNING! WARNING!  **\n");
-		printk(KERN_WARNING "**                                                        **\n");
-		printk(KERN_WARNING "** You are using 32-bit PTI on a 64-bit PCID-capable CPU. **\n");
-		printk(KERN_WARNING "** Your performance will increase dramatically if you     **\n");
-		printk(KERN_WARNING "** switch to a 64-bit kernel!                             **\n");
-		printk(KERN_WARNING "**                                                        **\n");
-		printk(KERN_WARNING "** WARNING! WARNING! WARNING! WARNING! WARNING! WARNING!  **\n");
-		printk(KERN_WARNING "************************************************************\n");
+		printk(KERN_INFO "\n");
+		printk(KERN_INFO "************************************************************\n");
+		printk(KERN_INFO "**                                                        **\n");
+		printk(KERN_INFO "** You are using 32-bit PTI on a 64-bit PCID-capable CPU. **\n");
+		printk(KERN_INFO "** Your performance will increase dramatically if you     **\n");
+		printk(KERN_INFO "** switch to a 64-bit install.                            **\n");
+		printk(KERN_INFO "**                                                        **\n");
+		printk(KERN_INFO "************************************************************\n");
 	}
 #endif
 
